cmd/vic-machine/create/create.go

```diff
@@ -665,53 +666,57 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 		return nil
 	}
 
-	log.Infof("### Installing VCH ####")
+	// create the logger for streaming VCH log messages
```

Comment:
can we have this fragment added for the vic-machine CLI calls to `configure` and `upgrade` as well - it's less significant for delete.

It'll be odd to have only the initial create call logged if we're to use it for service/audit.

Reply:
The logger is not currently written in a way that would allow it to be used in workflows other than create. I agree it should be generalized, but I don't think it's something we'll have time for in 1.3.